charts/asserts/templates/config/k8s-calls-rules-configmap.yaml

```diff
@@ -17,6 +18,15 @@ data:
               dst_workload: {{.Release.Name}}-server
               dst_namespace: {{.Release.Namespace}}
 
+          # asserts-ui -> authorization
+          - record: asserts:relation:calls
+            expr: sum by (asserts_env, asserts_site, namespace)
+                    (asserts:request:rate5m{job="{{.Release.Name}}-authorization", asserts_request_type="inbound"}) > 0
+            labels:
+              workload: {{.Release.Name}}-ui
+              dst_workload: {{.Release.Name}}-authorization
+              dst_namespace: {{.Release.Namespace}}
+
           # asserts-server -> postgres
```

Comment:
There's also `authorization` -> `postgres`

Reply:
I didn't add `authorization` -> `postgres` because there isn't a request metric that captures that connection (at least I could not find one).